extensions/amp-story-interactive/0.1/amp-story-interactive-slider.css

```diff
@@ -13,3 +13,42 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
+.i-amphtml-story-interactive-slider-range-container {
+  --background: white;
+  width: 100%;
+  background: var(--i-amphtml-interactive-option-background-color);
+  border-radius: 50px;
+  box-shadow: 2px 4px 8px rgba(0, 0, 0, 0.1);
+}
+.i-amphtml-story-interactive-slider-range-input {
+  -webkit-appearance: none;
+  width: calc(100% - 2em);
+  height: 1em;
+  margin: 1em;
+  background: transparent;
+  position: relative;
+  pointer-events: all;
+  cursor: pointer;
+}
+.i-amphtml-story-interactive-slider-range-input::before {
+  background: var(--interactive-accent-color);
+  content: '';
+  opacity: 0.3;
+  width: 100%;
+  height: 100%;
+  border-radius: 10px;
+  outline: none;
+  border: none;
+  position: absolute;
+  z-index: 2222;
+}
+.i-amphtml-story-interactive-slider-range-input::-webkit-slider-thumb {
+  -webkit-appearance: none;
+  width: 20px;
+  height: 20px;
+  border-radius: 50%;
+  background: var(--interactive-accent-color);
+  border: 1x solid var(--interactive-accent-color);
```

Comment:
We don't need a border for this AFAIK, we can just make the circle larger if we want